### PR TITLE
46-implement-data-saving-function

### PR DESCRIPTION
**Pull request by Elwizzy12**
@Elwizzy12  and @chendiblessing  developed the functionality to write scraped data to the chosen file format, amd 
Ensure data is saved in an organized and readable format.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -16,6 +19,12 @@ def index():
 def input():
     return render_template("input.html")
 
+@app.route("/sscrapy")
```
Comment (stephane-segning) [inline app.py:22]: Remove this one

File: blog_scrapped_data.csv
Comment (stephane-segning) [inline blog_scrapped_data.csv:1]: Remove this file from git please

File: app.py (hunk 1 above)
Comment (Elwizzy12) [reply app.py:22]: sir, this folder sscrapy consist of all the functions and it is routed in the app.py.
The oder folder scraping was deleted!
u can check the screenshot of what i'm talking about.

Screenshot from 2023-11-22 10-43-09

File: templates/base.html
Comment (stephane-segning) [inline templates/base.html:1]: Cancel these changes please

File: templates/input.html
Comment (stephane-segning) [inline templates/input.html:1]: Cancel these changes please

File: templates/results.html
Comment (stephane-segning) [inline templates/results.html:1]: Cancel these changes please

File: static/css/style.css
Comment (stephane-segning) [inline static/css/style.css:1]: Cancel these changes please

**Comment by stephane-segning** [comment]
Hello @Elwizzy12 @chendiblessing. You're doing well in proofing that you understood the tech. But you still have issues applying the changes required. So let me explain again.

1. Your ticket is just about implementing that method, who takes in a list of data scraped from websites, and then save them somewhere. Something like this:
   ```python
   def save_scrapped_data(data):
      #TODO <-- This is for you to do
      pass
   ```
2. You should also write a method to read what saved using the first one:
   ```python
   def read_scrapped_data(data):
      #TODO <-- This is for you to do too
      pass
   ```
3. You do nothing else inside of this ticket, as you're not required too. I'm not expecting then changes in `.html` files, nor `.css`,... But only in `.py` files.
4. Also, at most, I'm expecting a demo notebook file at most, for you to demonstrate the feature – that is if you want.

**Review by stephane-segning — CHANGES_REQUESTED** [review]
These changes does have a thing to do with this ticket. Please cancel them

File: scraping.py
Comment (stephane-segning) [inline scraping.py:1]: Your ticket is not to implement this. Cancel these changes please

File: app.py
Comment (stephane-segning) [inline app.py:1]: Your ticket is not to implement this. Cancel these changes please

File: .gitignore
Comment (stephane-segning) [inline .gitignore:1]: Your ticket is not to implement this. Cancel these changes please

**Review by stephane-segning — CHANGES_REQUESTED** [review]
These changes too should be cancelled, as they don't have anything to do with your ticket

**Comment by Elwizzy12** [comment]
@stephane-segning I have modified what you told me to do, please  review.

**Comment by stephane-segning** [comment]
@Elwizzy12 this is not a good implementation. Do you understand why, after the session we had today?

**Comment by Elwizzy12** [comment]
> @Elwizzy12 this is not a good implementation. Do you understand why, after the session we had today?

yes , you said you don't need another persons code to modify and implement your own ticket.
I sent it before we had the lesson yesterday!

**Pull request closed without merging**